### PR TITLE
Populate frontpage announcements

### PR DESCRIPTION
**Pull request by broad-well**
# Checklist

  * [x] This update has been tested thoroughly enough to ensure there are no
        functional bugs or obvious layout issues.
  * [x] This update has been tested in both desktop and mobile layouts.
  * [x] This update does not break any existing functionality, layout, or style
        rules, unless the update is explicitly intended to remove or update such
        functionality, layout, or style rules.
  * [x] This update is free of any code intended only for debugging or testing
        purposes.
  * [x] This update follows the FAMNM Website Code Style Guide.
  * [x] This update meets the visual style requirements as specified in the
        FAMNM Website Visual Style Guide, and any additional visual elements not
        previously specified in the FAMNM Website Visual Style Guide have been
        styled so as not to clash visually with the rest of the site.

# Description

This PR introduces a new layout for ephemeral announcements on the homepage and adds 3 announcements under this layout. The announcements are:
 - Edison bots are available for distribution to local youth robotics groups
 - The Ri3D team is featured in Michigan Engineering's college-wide news and social media
 - The new undergraduate program in Robotics has been approved and will be available for declaration in Fall 2022

**Preview the changes [here](https://dev.famnm-website.pages.dev).**

### CODE REVIEW

**Review by yottalogical — APPROVED** [review]

**Comment by broad-well** [comment]
I had this concern as well. Our consensus prefers to have the logo and definition situated vertically rather than horizontally, which slightly increases the vertical space used before the announcements. This means our only way to remediate this is to modify the carousel, for which I'd like to get the core team's approval (as it's the website's most prominent feature). I will merge this for now to publish the announcements. Changes to the carousel will appear as a separate PR.

**Pull request merged**
